src/common/main.lua

```diff
@@ -515,7 +515,7 @@ function ant.init()
 			end
 
 			local srcCodeTxIdStatus, srcCodeTxIdResult = pcall(utils.validateArweaveId, srcCodeTxId)
-			if srcCodeTxIdStatus and not srcCodeTxIdStatus then
+			if srcCodeTxIdStatus and not srcCodeTxIdResult then
```

Comment:
this was caught by the linter - so I modified to what I thought made sense. now it's failing evolve tests.

@atticusofsparta - can you take a look?

Reply:
looks like this should be [7b54a93](https://github.com/ar-io/ar-io-ant-process/pull/29/commits/7b54a93cfb1b220404b16027086e8cda58415511)